app/components/visualizations/scatter-plot.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  init() {
+    this._super(...arguments);
+  },
+
+  // Set html elements
+  tagName: "div",
+  classNames: ["gViz-wrapper"],
+
+  // Set unique _id
+  _id: `visualization-${Math.floor(Math.random() * (1000000000 - 5 + 1)) + 5}`,
```

Comment:
You don't need to do this, since ember is responsible for creating an uniq id here